R/prepare-admin-panel.R

```diff
@@ -125,14 +125,54 @@ get_per_day_plot_data <- function(base, per_day) {
 prepare_admin_panel_components <- function(
   input, output, session, data_storage
 ) {
+  if (nrow(data_storage$read_event_data()) == 0) {
+    shiny.semantic::create_modal(
+      shiny.semantic::modal(
+        id = "no_data_modal",
+        title = "Warning",
+        content = shiny::tags$div(
+          class = "grid",
+          shiny::tags$div(
+            class = "sixteen wide column center aligned",
+            style = "text-align: center; padding: 2rem 0;",
+            shiny::tags$div(
+              class = "ui icon header red",
+              shiny::tags$i(class = "exclamation triangle icon"),
+              shiny::tags$div(
+                class = "content",
+                "No telemetry data available!!",
+                shiny::tags$div(
+                  class = "sub header",
+                  "Please run the Instrumentation app first to generate some data"
```

Comment:
Maybe change here from "Instrumentation" to "app with `shiny.telemetry`" or a variation that makes sense.

I don't think "Instrumentation" is the right word here. WDYT?

Reply:
I think it makes sense. Users may not understand what `Instrumentation` app is.
How about "Please run an app with `shiny.telemetry` first to generate some data"?

Reply:
Alos thinking of making the `shiny.telemetry` a clickable link that will take the user to the documentation page.
What do you think?

Reply:
Gretat idea!! :bulb:

Reply:
Probably with  a `target="_blank"` so it opens on a new tab